modules/tf_cloudbuild_builder/variables.tf

```diff
@@ -94,12 +93,24 @@ variable "trigger_name" {
 variable "trigger_location" {
   description = "Location of the Cloud Build trigger building the Terraform builder. If using private pools should be the same location as the pool."
   type        = string
-  default     = "global"
```

Comment:
Do we need to remove the default?  Looking if we can make this no-longer a breaking change.

Reply:
Hi @apeabody , Cloud build Repositories (2nd Gen) [does not support](https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#:~:text=Note%3A%20You%20must%20specify%20a%20region.%20Your%20connection%20cannot%20exist%20globally.) the `global` location, that was the reason for removing it. 
We can keep the default `global`, but it is not valid for the repository types that will replace CSR and the user will need to provide a different value.

Reply:
Got it - Thanks @daniel-cit!  I think we can either leave and ideally address in some other fashion, or document the breaking change similar to https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/docs/upgrading_to_v5.0.md (next version will be v9.0)

Reply:
fixed